templates/includes/navbar.html

```diff
@@ -1,82 +1,82 @@
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="navbar-collapse collapse" id="navbar">
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item">
+                    <a class="nav-link" href="{% url 'home' %}">Home</a>
+                </li>
+                {% if request.user.user_type|is_types:authorised_admins %}
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" id="navbarHackathons" role="button" data-toggle="dropdown"
+                        aria-haspopup="true" aria-expanded="false">
+                        Hackathons
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarHackathons">
+                        <!-- TODO: Separate between different admin users -->
+                        <a class="dropdown-item" href="{% url 'hackathon:create_hackathon' %}">Create Hackathon</a>
+                        <a class="dropdown-item" href="{% url 'hackathon:hackathon-list' %}">View Hackathons</a>
+                    </div>
+                </li>
+                {% else %}
+                <li class="nav-item">
+                    <a class="nav-link" href="{% url 'hackathon:hackathon-list' %}">Hackathons</a>
+                </li>
+                {% endif %}
+                {% if user.is_authenticated %}
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" id="navbarResources" role="button" data-toggle="dropdown"
+                        aria-haspopup="true" aria-expanded="false">
+                        Resources
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarResources">
+                        <a class="dropdown-item" href="{% url 'faq' %}">FAQs</a>
+                        <a class="dropdown-item" href="{% url 'resources' %}">Useful Resources</a>
+                    </div>
+                </li>
+                {% else %}
+                <li class="nav-item">
+                    <a class="nav-link" href="{% url 'faq' %}">FAQs</a>
+                </li>
+                {% endif %}
+                <li class="nav-item">
+                    <a class="nav-link" href="{% url 'view_showcases' %}">Project Showcase</a>
+                </li>
+                <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" id="navbarAccount" role="button" data-toggle="dropdown"
+                        aria-haspopup="true" aria-expanded="false">
+                        My Account
+                    </a>
+                    <div class="dropdown-menu" aria-labelledby="navbarAccount">
+                        {% if not user.is_authenticated %}
+                            {% if not socialaccount_providers%}
```

Comment:
Missing _space_ between variable and `%}`

Reply:
Added missing space.